halo/cmd/flags.go

```diff
@@ -25,3 +25,10 @@ func bindInitFlags(flags *pflag.FlagSet, cfg *InitConfig) {
 	flags.BoolVar(&cfg.Force, "force", cfg.Force, "Force initialization (overwrite existing files)")
 	flags.BoolVar(&cfg.Clean, "clean", cfg.Clean, "Delete home directory before initialization")
 }
+
+func bindOperatorFlags(flags *pflag.FlagSet, cfg *OperatorConfig) {
+	libcmd.BindHomeFlag(flags, &cfg.HomeDir)
+	flags.StringVar(&cfg.L1ChainName, "l1chain", cfg.L1ChainName, "The layer 1 chain name")
+	flags.StringVar(&cfg.AVSDirectoryAddr, "avs-dir-addr", cfg.AVSDirectoryAddr, "The AVS Directory contract address")
```

Comment:
We can read the AVS direcetory from the OmniAVAS contract, so operators don't need to find the AVS directory address